docarray/array/storage/weaviate/find.py

```diff
@@ -75,12 +91,20 @@ def _find_similar_vectors(
 
             doc.tags['wid'] = result['_additional']['id']
 
+            if additional:
+                for add in additional:
+                    doc.tags[f'{add}'] = result['_additional'][add]
+
             docs.append(doc)
 
         return DocumentArray(docs)
 
     def _filter(
-        self, filter: Dict, limit: Optional[Union[int, float]] = 20
+        self,
+        filter: Dict,
+        limit: Optional[Union[int, float]] = 20,
+        additional: Optional[Union[Dict, List]] = None,
```

Comment:
please add docstrings to the new parameters

Reply:
Docstrings are added